<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>742</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Overpass 2 GPX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ausgabedatei:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelAusgabedatei">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>5</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Keine Datei ausgewählt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDateiauswahl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="statusTip">
         <string>Ausgabedatei auswählen...</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Auswählen...</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditAuthor">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Author(Name &amp; Email)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>GPX Beschreibung</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditgpxdescription">
      <property name="toolTip">
       <string>Was für Daten sin din der GPX-Datei vorhanden</string>
      </property>
      <property name="placeholderText">
       <string>Beschreibung</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Copyright</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCopyright">
      <property name="placeholderText">
       <string>Copyright</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Website</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditdefgpxwebsite">
      <property name="placeholderText">
       <string>URL</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Overpass Abfrage:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="textEditAbfrage">
      <property name="placeholderText">
       <string>Bitte OverpassQL Abfrage eingeben</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBarWriteGPX">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonAbfrage">
      <property name="text">
       <string>Ausführen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNeu"/>
    <addaction name="actionOeffnen"/>
    <addaction name="actionSpeichern"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNeu">
   <property name="text">
    <string>Neu...</string>
   </property>
  </action>
  <action name="actionOeffnen">
   <property name="text">
    <string>Öffnen...</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
